include/psp2kern/net/net.h

```diff
@@ -706,10 +706,14 @@ int ksceNetSocket(const char *name, int domain, int type, int protocol);
 int ksceNetAccept(int s, SceNetSockaddr *addr, unsigned int *addrlen);
 int ksceNetBind(int s, const SceNetSockaddr *addr, unsigned int addrlen);
 int ksceNetListen(int s, int backlog);
-int ksceNetRecv(int s, void *buf, unsigned int len, int flags);
+int ksceNetRecv(int s, void *buf, unsigned int len, int flags); //!< This function does not actually exist in Kernel, so it's a #define to ksceNetRecvfrom() for compatibility
 int ksceNetRecvfrom(int s, void *buf, unsigned int len, int flags, SceNetSockaddr *from, unsigned int *fromlen);
-int ksceNetSend(int s, const void *msg, unsigned int len, int flags);
+int ksceNetSend(int s, const void *msg, unsigned int len, int flags); //!< This function does not actually exist in Kernel, so it's a #define to ksceNetSendto() for compatibility
```

Comment:
The `#define` section is further down, so I thought that it would be more clear for doc readers to leave the function prototype. Do you want me to remove it?